addons/prometheus/0.48.1-16.10.0/install.sh

```diff
@@ -88,7 +94,11 @@ function grafana_admin_secret() {
 function prometheus_outro() {
     printf "\n"
     printf "\n"
-    printf "The UIs of Prometheus, Grafana and Alertmanager have been exposed on NodePorts ${GREEN}30900${NC}, ${GREEN}30902${NC} and ${GREEN}30903${NC} respectively.\n"
+    if [ -z "$PROMETHEUS_SERVICE_TYPE" ] || [ "$PROMETHEUS_SERVICE_TYPE" = "NodePort" ] ; then
+        printf "The UIs of Prometheus, Grafana and Alertmanager have been exposed on NodePorts ${GREEN}30900${NC}, ${GREEN}30902${NC} and ${GREEN}30903${NC} respectively.\n"
+    else
+        printf "The UIs of Prometheus, Grafana and Alertmanager have been exposed on internal ClusterIP services.\n"
```

Comment:
Is there a reason Grafana is included in this? It seems like the obvious frontend to these services, and requires authentication. Although it would also make sense to configure TLS on it.